language-design.md

```diff
@@ -59,7 +138,16 @@ What does the post on grayscale tell us about the process of API design?
 
 **Response**
 
-
+There are going to be multiple naming options that make sense for most, if not
+all API functions. Since the post says "we" need your feedback, it's likely that
+there were a team of people unable to decide the best option. The voting poll
+makes it clear that the input of the potential user base and programmers in 
+general matters a lot in making the decision.  What is also interesting is the
+comments section, where several additional ideas and opinions were 
+expressed by readers, showing the interest in the topic and how many different
+Opinions there are regarding intuitiveness of the name of a grayscale function. 
+This also shows that naming is extremely important for APIs, so much so that 
+blog posts and voting polls for a single function's name is appropriate.
 
```

Comment:
I was also surprised by the care that the API designers demonstrated by polling users and the thoughtful discussion about the function's name.  To add to your response, my biggest takeaway was that, as an API designer, you're not going to please everybody.  All the methods clearly had their drawbacks, and there was no consensus.